kolibri/content/api.py

```diff
@@ -4,221 +4,173 @@
 """
 from functools import wraps
 
-from django.core.files import File as DjFile
 from django.db.models import Q
 from kolibri.content import models as KolibriContent
 from kolibri.content.utils import validate
+from .constants import content_kinds
 
 """ContentDB API methods"""
 
 def can_get_content_with_id(func):
     """
-    decorator function for returning ContentMetadata object when giving content id.
+    decorator function for returning ContentNode object when giving content id.
```

Comment:
Can you explain this function's algorithm? I find the docstring to not be aligned with the function name as well:

`can_get_content_with_id` implies that it returns a boolean function, or that it checks permissions somehow.

`decorator function for returning ContentNode object` implies that we simplie return a ContentNode object.